src/components/partials/IdentitySearch/IdentitySearch.vue

```diff
@@ -149,10 +149,8 @@ export default class IdentitySearch extends Vue {
             this.selectedChainKey
         );
         if (!this.implementsPallet) {
-            // TODO: show nice error partial component instead of standard alert
-            alert(
-                "Sorry, the selected node is not available or does not implement the identity pallet"
-            );
+            const message = "Sorry, the selected node is not available or does not implement the identity pallet";
```

Comment:
`this.implementsPallet = await this.store.dispatch(
            "IDENTITY_PALLET_EXISTS",
            this.selectedChainKey
        );
`
Should be in try catch. In the catch the message for the error should be "Sorry, the connection to the node could not be established"
and if it does not implement the identity pallet -> "Sorry, the selected node does not implement the identity pallet"